src/snowflake/snowpark/functions.py

```diff
@@ -1173,7 +1204,21 @@ def degrees(e: ColumnOrName) -> Column:
 
 
 def radians(e: ColumnOrName) -> Column:
-    """Converts degrees to radians."""
+    """Converts degrees to radians.
+
+    Examples::
+
+        >>> df = session.create_dataframe([[1.111], [2.222], [3.333]], schema=["a"])
+        >>> df.select(radians(col("a")).alias("result")).show()
```

Comment:
Could we use a `CAST` to limit the number of digits after the floating point here?

Reply:
updated, how does it look to you now?